README.md

```diff
@@ -514,7 +514,7 @@ Thus when you create a TODO, it is almost always your name that is given.
      end
      ```
 
-* <a name="no-default-args"></a>Do not use default arguments. Use an options
+* <a name="no-default-args"></a>Do not use default positional arguments. Use named arguments or an options
```

Comment:
What is the reason we want to allow default keyword args, but not default positional args? I think one reason behind discouraging default args was to remove the obscurity around them and make it explicit from a caller's standpoint what is being invoked.